src/p2p/net_node.inl

```diff
@@ -354,14 +374,14 @@ namespace nodetool
       {
         epee::net_utils::network_address na{epee::net_utils::ipv4_network_address{boost::asio::detail::socket_ops::host_to_network_long(endpoint.address().to_v4().to_ulong()), endpoint.port()}};
         seed_nodes.push_back(na);
-        MINFO("Added seed node: " << na.str());
+        MINFO("Added node: " << na.str());
       }
       else
       {
         MWARNING("IPv6 unsupported, skip '" << host << "' -> " << endpoint.address().to_v6().to_string(ec));
-        throw std::runtime_error("IPv6 unsupported");
```

Comment:
I removed this because erroring out here seems not right given the above warning message.

Reply:
I see no need to make it harder to fix things when non IPv4 gets added. Why do you think it is unnecessary ?

Reply:
For example, monerohash.com resolves to both IPv4 and IPv6, so this exception will cause the daemon to halt:

```
build/release/bin/monerod --log-level net.p2p:INFO --add-priority-node monerohash.com
2018-06-11 07:34:01.246	  0x7fffc9e663c0	INFO 	net.p2p	src/p2p/net_node.inl:360	Resolving node address: host=monerohash.com, port=18080
2018-06-11 07:34:01.377	  0x7fffc9e663c0	INFO 	net.p2p	src/p2p/net_node.inl:377	Added node: 107.191.99.227:18080
2018-06-11 07:34:01.377	  0x7fffc9e663c0	INFO 	net.p2p	src/p2p/net_node.inl:377	Added node: 107.191.99.95:18080
2018-06-11 07:34:01.377	  0x7fffc9e663c0	WARN 	net.p2p	src/p2p/net_node.inl:381	IPv6 unsupported, skip 'monerohash.com' -> 2604:180:2:141e::47d9
```

Should the exception be caught in some ways instead of deleting this line?

Reply:
OK, that's a good reason.